fix download modal buttons

kubic71 opened the pull request.

kubic71 commented:
On desktop:
![image](https://github.com/jMyles/VowelSoundsAlbumDrop/assets/12272640/054b7e18-86fa-410a-b7c5-160525dc5023)

Mobile:
![image](https://github.com/jMyles/VowelSoundsAlbumDrop/assets/12272640/b9be53d9-8213-4b5f-8b2b-3511084e1557)

KPrasch reviewed (approved):
This appears to fix the overflow/wrapping issue on my phone.  LGTM ⚡

kubic71 commented:
@KPrasch I also just fixed the color theme of the buttons in the modal.
Also added a link to each button (because you said "buttons all thud (lead to nowhere without indication)")

Maybe a re-review?

kubic71 commented:
![image](https://github.com/jMyles/VowelSoundsAlbumDrop/assets/12272640/cb6e0fdb-09f9-4a76-b449-4a2c41bd9b98)

kubic71 commented:
![ezgif-2-90c2b77348](https://github.com/jMyles/VowelSoundsAlbumDrop/assets/12272640/8a1c101c-4155-4e2b-818a-8c4cdf767ffb)

kubic71 commented:
Note: I also switched the order of css files included, and put our `style.css` after the Bootstrap.
  That makes more sense to me, as it allows us to customize the Boostrap styles more easily, without the need to resort to using `!important` for instance.

KPrasch commented:
Okay, some time has passed here and this will need a rebase to be integrated.

kubic71 commented on src/index.html:
```diff
@@ -82,19 +82,16 @@ <h1 id="dateAnnouncement">
                 <div class="modal-content">
                     <div class="modal-header">
                         <h5 class="modal-title" id="exampleModalLabel">Get Vowel Sounds, Encrypted</h5>
-                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
```
Removed this, beause we already have a dedicated "close" button, and this  extra button looked off-theme color-wise.

kubic71 commented on src/index.html:
```diff
@@ -82,19 +82,16 @@ <h1 id="dateAnnouncement">
-                        <button type="button" class="btn btn-primary">IPFS</button>
-                        <button type="button" class="btn btn-primary">Arweave</button>
-                        <button type="button" class="btn btn-primary">Bittorrent</button>
+                        <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
```
Turns out we need `data-bs-dismiss` instead of `data-dismiss` with the current bootstrap version

kubic71 commented:
Rebase onto the current main and should be working again :)

jMyles commented:
Fixed #63
